backend/src/auth/config.rs

```diff
@@ -332,6 +332,10 @@ pub(crate) struct RoleConfig {
     #[config(default = "ROLE_TOBIRA_UPLOAD")]
     pub(crate) upload: String,
 
+    /// The role granting permission to use Tobira's Opencast API.
+    #[config(default = "ROLE_TOBIRA_CAN_CREATE_SERIES")]
+    pub(crate) can_create_series: String,
```

Comment:
The description is a bit too broad right? "Tobira's Opencast APIs"? Why not simply "... to create new series through Tobira"?